controller/src/main.rs

```diff
@@ -81,59 +84,48 @@ impl SchedulingService for SchedulerService {
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
-    // // Initialize logger
+    // SETUP LOGGING
```

Comment:
Why are we changing perfectly fine comments to new, upper case ones?

Reply:
done